src/style.css

```diff
@@ -0,0 +1,67 @@
+html,
+body {
+  margin: 0;
+}
+
+.frame {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  left: 600px;
+  top: 150px;
+  width: 140px;
+  height: 160px;
+}
+
+.stars-frame {
+  width: 96px;
+  height: 96px;
+}
+
+.stars {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
```

Comment:
Add right margin for stars except last one

Reply:
> Add right margin for stars except last one

I don't understand, I trying to add but it's still wrong..